pytest_html/plugin.py

```diff
@@ -324,9 +325,13 @@ def _save_report(self, report_content):
         if not os.path.exists(os.path.dirname(self.logfile)):
             os.makedirs(os.path.dirname(self.logfile))
         logfile = open(self.logfile, 'w', encoding='utf-8')
+        style_file = open('style.css', 'w', encoding='utf-8')
```

Comment:
This puts the `style.css` in the working directory, which means if the HTML report is stored anywhere else the CSS file will not be found. We should be sure to always place it alongside the HTML file.